strategies.py

```diff
@@ -53,3 +54,31 @@ def search(self, board: chess.Board, *args: Any) -> PlayResult:
         moves = list(board.legal_moves)
         moves.sort(key=str)
         return PlayResult(moves[0], None)
+
+
+class ComboEngine(ExampleEngine):
+    """Get a move using multiple different methods."""
```

Comment:
I think it's important to mention that this demonstrates a `search()` method that can use time limits and other game information.